reference/bundles/custom_entity_bundle/abstract_entities_and_repositories.rst

```diff
@@ -0,0 +1,25 @@
+Abstract Entities and Repositories
+==================================
+
+The bundle provides a series of abstract entities.
+
+Entity\\AbstractCustomEntity
+----------------------------
+
+This entity implements the basic interfaces that are used in Akeneo. It defines a "code" property which is
+used throughout the PIM as the entity's identifier.
+
+The repositories for this type of entities should extend
+``Pim\Bundle\CustomEntityBundle\Entity\Repository\CustomEntityRepository``.
+
+
+Entity\\AbstractTranslatableCustomEntity
+----------------------------------------
+
+This entity extends the AbstractCustomEntity, and provides a link to a translation entity. This is used
+to provide a different label or description for example for each activated locale.
```

Comment:
`for example for each activated locale.` => for each activated locale for instance ?